File: test/fixtures/sample.js
function add(a, b) {
  return a + b;
}

class Point {
  constructor(x, y) {
    this.x = x;
    this.y = y;
  }

  distance(other) {
    const dx = this.x - other.x;
    const dy = this.y - other.y;
    return Math.sqrt(dx * dx + dy * dy);
  }
}

const sum = add(1, 2);
const p1 = new Point(0.0, 0.0);
const p2 = new Point(3.0, 4.0);
console.log(`Sum: ${sum}`);
console.log(`Distance: ${p1.distance(p2)}`);

File: test/fixtures/sample.py
import math

def add(a, b):
    return a + b

class Point:
    def __init__(self, x, y):
        self.x = x
        self.y = y

    def distance(self, other):
        dx = self.x - other.x
        dy = self.y - other.y
        return math.sqrt(dx * dx + dy * dy)

sum_result = add(1, 2)
p1 = Point(0.0, 0.0)
p2 = Point(3.0, 4.0)
print(f"Sum: {sum_result}")
print(f"Distance: {p1.distance(p2)}")
